pkg/csi_driver/node_test.go

```diff
@@ -177,11 +188,54 @@ func TestNodePublishVolume(t *testing.T) {
 			},
 			expectErr: true,
 		},
+		// TODO: enable this test when FakeMount supports Windows behavior
```

Comment:
It might be cleaner to actually have a separate test function just for the windows cases.

Reply:
Done.

Reply:
Is there something still preventing this test from being enabled?  If so, let's open an issue for it and reference it here

Reply:
On Windows, if TargetPath exists it should return an error but FakeMount doesn't do that.

Reply:
Opened an issue on K/K: https://github.com/kubernetes/kubernetes/issues/81609